list.js

```diff
@@ -1,4 +1,4 @@
-/* global $, jQuery */
+/* global jQuery, jQuery */
```

Comment:
don't need to tell eslint that `jQuery` is global twice - this can be changed back to `$`